tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "forceConsistentCasingInFileNames": true,
     "strict": true,
     "noImplicitOverride": true,
-    "noPropertyAccessFromIndexSignature": true,
+    "noPropertyAccessFromIndexSignature": false,
```

Comment:
We are disabling this feature because it hinders the development, like when we developed the input validators to display error message, it keeps complaining about signature, that we have to use bracket notation, e.g.: `errors?.email['minlength']` instead of dot notation, e.g.: `errors?.email?.minlength`